src/ui/ResultLink/ResultLink.ts

```diff
@@ -246,8 +246,11 @@ export class ResultLink extends Component {
     public os?: OS_NAME
   ) {
     super(element, ResultLink.ID, bindings);
-    this.options = ComponentOptions.initComponentOptions(element, ResultLink, options);
-    this.options = _.extend({}, this.options, this.componentOptionsModel.get(ComponentOptionsModel.attributesEnum.resultLink));
+    this.options = {
```

Comment:
Hum... I guess it's okay, but there are some differences between _.extend and es6 spread operator.

For example, underscore js is "pretty good" at dealing with `null/undefined`, as opposed to es6 spread which I think would give a different output.

Reply:
Good point, I will revert to using underscore